source/reference/minio-mc.rst

```diff
@@ -233,6 +233,37 @@ The following table lists :mc-cmd:`mc` commands:
           :start-after: start-mc-head-desc
           :end-before: end-mc-head-desc
      
+   * - | :mc:`mc idp ldap add`
+       | :mc:`mc idp ldap disable`
+       | :mc:`mc idp ldap enable`
+       | :mc:`mc idp ldap info`
+       | :mc:`mc idp ldap ls`
+       | :mc:`mc idp ldap policy`
+       | :mc:`mc idp ldap rm`
+       | :mc:`mc idp ldap update`
+     - .. include:: /reference/minio-mc/mc-idp-ldap.rst
+          :start-after: start-mc-idp-ldap-desc
+          :end-before: end-mc-idp-ldap-desc
+
+   * - | :mc:`mc idp ldap add`
+       | :mc:`mc idp ldap disable`
+       | :mc:`mc idp ldap enable`
+       | :mc:`mc idp ldap info`
+       | :mc:`mc idp ldap ls`
+       | :mc:`mc idp ldap policy`
```

Comment:
Should this be `policy` or `policy entities`?

The existing pages have `mc admin idp ldap policy entities` as a subcommand of both `mc admin idp ldap` and `mc admin idp ldap policy`. (Which are siblings.)

It seems to make sense to show `policy` as a subcommand of `mc idp ldap`, that page already has all three `policy` subcommands. That would suggest moving `mc idp ldap policy` under `mc idp ldap`. Although I'm not aware why `mc admin idp ldap policy entities` was on the `ldap` page rather than a child one.

Reply:
Since there are three subcommands under `mc admin idp ldap|oidc policy`, we might as well just say `policy` here

If/when we split those commands out we'll have to revisit this.